test/controllers/api/v2/packages_controller_test.rb

```diff
@@ -82,6 +82,15 @@ def test_index_with_available_for_content_view_version
       assert_includes ids, @rpm.id
     end
 
+    def test_index_with_search_relative_release
+      response = get :index, params: { :search => 'release > 1.el7' }
+
+      assert_response :success
+      ids = JSON.parse(response.body)['results'].map { |p| p['id'] }
+      assert_includes ids, katello_rpms(:one_two).id
+      assert_includes ids, katello_rpms(:three).id
+    end
```

Comment:
putting this in rpm_test.rb would be a better place, and if you could add more searches there, that'd be great.  I don't see any searches in rpm_test.rb, but here's what one would look like:

https://github.com/Katello/katello/blob/master/test/models/erratum_test.rb#L27-L29